### DISCUSSION TITLE
Changing / adding -env on run container?

### DISCUSSION THREAD

**Discussion by Christolito** [lloesche/valheim-server-docker#540] [Q&A]
How do I update or add new -env that I want to pass? Do I have to delete the container and run it again? Will it overwrite or mess up the current world? (If I use the same name etc when defining it in the run command) THX btw - im running this on Mac OS Docker desktop - works flawlessly + I've learn a lot about Docker 👍

**Comment by dylanReinsma** [comment]
stop the container, go to settings of container and add it, start container
